Formula/geckodriver.rb

```diff
@@ -1,8 +1,10 @@
 class Geckodriver < Formula
   desc "WebDriver <-> Marionette proxy"
   homepage "https://github.com/mozilla/geckodriver"
-  url "https://github.com/mozilla/geckodriver/archive/v0.25.0.tar.gz"
-  sha256 "9ba9b1be1a2e47ddd11216ce863903853975a4805e72b9ed5da8bcbcaebbcea9"
+  # Get the commit id for stable releases from https://github.com/mozilla/geckodriver/releases
+  url "https://hg.mozilla.org/mozilla-central/archive/e9783a644016aa9b317887076618425586730d73.tar.gz"
```

Comment:
This url returns an error 
```
curl: (22) The requested URL returned error: 403 Archive type not allowed: gz
Error: An exception occurred within a child process:
  DownloadError: Failed to download resource "geckodriver"
Download failed: https://hg.mozilla.org/mozilla-central/archive/e9783a644016aa9b317887076618425586730d73.tar.gz
```

Reply:
Thanks for reporting this. Looks like the only allowed archive type is now `.zip`. Working on a fix.

Reply:
np @chrmoritz was a bit frustrating after the rust dependency compiling for 2-3 hours, and then this step failing... would be also great to have just `rustup` being called instead of having to compile rust from source - but no idea where that's happening

Reply:
Are you on an unsupported version of macOS (<= Sierra)? If not, you could also use homebrew's bottles for `rust` (and `geckodriver`) if you are considering consuming upstreams `rust` binaries via `rustup`?